components/automate-ui/src/app/pages/+compliance/shared/reporting/report-query.service.ts

```diff
@@ -44,15 +44,21 @@ export class ReportQueryService {
   }
 
   getReportQuery(): ReportQuery {
-    return this.state.getValue();
+    const reportQuery = this.state.getValue();
+    return {
+      startDate: reportQuery.startDate.clone(),
+      endDate: reportQuery.endDate.clone(),
+      interval: reportQuery.interval,
+      filters: Object.assign([], reportQuery.filters)
+    };
```

Comment:
Returning a clone of the current state to avoid the state changing unintentionally